### PR TITLE
experience/13606/daily-data-for-all

### PR DESCRIPTION
**Pull request by etanb**
Switch all users to see Daily Data instead of Data Dashboard, including content updates

Fixes: #13606

### CODE REVIEW

File: frontend-react/src/pages/deliveries/Deliveries.tsx
Comment (chris-kuryak) [inline frontend-react/src/pages/deliveries/Deliveries.tsx:1]: Also remove the line "You can find additional detail for COVID..."

File: frontend-react/src/content/managing-your-connection/index.mdx [hunk 1]
```diff
@@ -74,14 +74,14 @@ import site from "../../content/site.json";
                     </Card>
                     <Card gridLayout={{col: "fill"}} isAlternate>
                         <CardMedia>
-                            <img src="/assets/img/DataDashboard.png" alt="Data Dashboard" />
+                            <img src="/assets/img/DailyData.png" alt="Data Dashboard" />
```
Comment (penny-lischer) [inline frontend-react/src/content/managing-your-connection/index.mdx:77]: Change alt test too

**Review by penny-lischer — APPROVED** [review]
Small change, otherwise LGTM

**Review by audreykwr — CHANGES_REQUESTED** [review]
Only thing is the same thing you noticed and that Chris commented on re: that lingering Data Dash reference on Daily Data.

**Comment by etanb** [comment]
it has been changed already in this commit: [8813b4f](https://github.com/CDCgov/prime-reportstream/pull/14017/commits/8813b4fe988430621029c27e0b79bc8c1403c7d0)

**Comment by etanb** [comment]
@audreykwr it has been changed already in this commit: [8813b4f](https://github.com/CDCgov/prime-reportstream/pull/14017/commits/8813b4fe988430621029c27e0b79bc8c1403c7d0)

File: frontend-react/src/content/managing-your-connection/index.mdx (hunk 1 above)
Comment (etanb) [reply frontend-react/src/content/managing-your-connection/index.mdx:77]: good catch!

**Review by audreykwr — APPROVED** [review]

**Pull request merged**